spec/spec_helper.rb

```diff
@@ -20,7 +20,12 @@
 
 # Ensure the Active Admin load path is happy
 require 'rails'
+
 require 'active_model'
+
+# require ActiveRecord to ensure that Ransack loads correctly
+require 'active_record'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.